torchvision/models/squeezenet.py

```diff
@@ -71,12 +69,6 @@ def __init__(self, version='1_0', num_classes=1000):
                 Fire(384, 64, 256, 256),
                 Fire(512, 64, 256, 256),
             )
-        else:
```

Comment:
I think it is worthwhile to keep this.
For example, if the user passes `1.0` instead of `1_0`, this will silently dispatch to the `1_1` implementation.